homeassistant/components/device_tracker/mqtt_json.py

```diff
@@ -0,0 +1,88 @@
+"""
+Support for GPS tracking MQTT enabled devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.mqtt_json/
+"""
+import asyncio
+import json
+import logging
+
+import voluptuous as vol
+
+import homeassistant.components.mqtt as mqtt
+from homeassistant.core import callback
+from homeassistant.const import CONF_DEVICES
+from homeassistant.components.mqtt import CONF_QOS
+from homeassistant.components.device_tracker import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+
+DEPENDENCIES = ['mqtt']
+
+_LOGGER = logging.getLogger(__name__)
+
+LAT_KEY = 'lat'
```

Comment:
Will do, please note that [device_tracker/__init__.py](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/__init__.py#L71) is not using ATTR_BATTERY_LEVEL.